scripts/util.cjs

```diff
@@ -469,7 +469,8 @@ const generateDappListsForChain = async (chain) => {
 
     // Remove socialLinks and fullDescription from each dApp object
     dapps.results = dapps.results.map((dapp) => {
-      const { socialLinks, fullDescription, ...filteredDapp } = dapp
+      const { socialLinks, fullDescription, logo, ...filteredDapp } = dapp
+      filteredDapp.logo = logo.replace('dashboard-assets.dappradar.com', 'dashboard-assets-dappradar.wallet.brave.com')
```

Comment:
for security reasons it's better to parse the URL and just replace the hostname, so that a url like `https://notdashboard-assets.dappradar.com` or `https://example.com/?query=dashboard-assets.dappradar.com` won't get mangled